standards/ICRC-3/README.md

```diff
@@ -86,238 +86,661 @@ An ICRC-3 compliant Block
+      "tx";
+      variant {
+        Map = vec {
+          record { "amt"; variant { Nat64 = 500_000_000 : nat64 } };
+          record {
+            "to";
+            variant {
+              Array = vec {
+                variant { Blob = blob "\15\28\84\12\af\11\b2\99\31\3a\5b\5a\7c\12\83\11\de\10\23\33\c4\ad\be\66\9f\2e\a1\a3\08" };
+              }
+            };
+          };
+          record { "op"; variant { Text = "mint" } };
+        }
+      };
+    };
+  }
+};
 ```
 
-#### Transfer and Transfer From Block Schema
+---
 
-1. the `btype` field MUST be
-    1. `"2xfer"` for `icrc2_transfer_from` blocks
-    2. `"1xfer"` for `icrc1_transfer` blocks
-1. if `btype` is not set then `tx.op` field MUST be `"xfer"`
-2. it MUST contain a field `tx.from: Account`
-3. it MUST contain a field `tx.to: Account`
-4. it CAN contain a field `tx.spender: Account`
+#### Example 3: Burn from user account
+This example represents an `icrc1_transfer` call where the destination `to` is the minting account. This results in a burn block. The caller specifies `from` and `amount`. No `fee`, `memo`, or `created_at_time` are provided.
 
-Example with `btype`:
 ```
-variant { Map = vec {
-    record { "btype"; "variant" { Text = "1xfer" }};
-    record { "fee"; variant { Nat = 10 : nat } };
-    record { "phash"; variant { Blob =
-        blob "h,,\97\82\ff.\9cx&l\a2e\e7KFVv\d1\89\beJ\c5\c5\ad,h\5c<\ca\ce\be"
-    }};
-    record { "ts"; variant { Nat = 1_701_109_006_692_276_133 : nat } };
-    record { "tx"; variant { Map = vec {
-        record { "amt"; variant { Nat = 609_618 : nat } };
-        record { "from"; variant { Array = vec {
-                variant { Blob = blob "\00\00\00\00\00\f0\13x\01\01" };
-                variant { Blob = blob "\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00" };
-        }}};
-        record { "to"; variant { Array = vec {
-            variant { Blob = blob " \ef\1f\83Zs\0a?\dc\d5y\e7\ccS\9f\0b\14a\ac\9f\fb\f0bf\f3\a9\c7D\02" };
-            variant { Blob = blob "\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00" };
-        }}};
-    }}};
-}};
+variant {
+  Map = vec {
+    record {
+      "phash";
+      variant {
+        Blob = blob "\7f\89\42\a5\be\4d\af\50\3b\6e\2a\8e\9c\c7\dd\f1\c9\e8\24\f0\98\bb\d7\af\ae\d2\90\10\67\df\1e\c1\0a"
+      };
+    };
+    record { "ts"; variant { Nat64 = 1_753_344_740_000_000_000 : nat64 } };
+    record {
+      "tx";
+      variant {
+        Map = vec {
+          record { "amt"; variant { Nat64 = 42_000_000 : nat64 } };
+          record {
+            "from";
+            variant {
+              Array = vec {
+                variant { Blob = blob "\00\00\00\00\02\30\02\17\01\01" };
+              }
+            };
+          };
+          record { "op"; variant { Text = "burn" } };
+        }
+      };
+    };
+  }
+};
 ```
 
-Example without `btype`:
+##### `icrc2_transfer_from`
```

Comment:
Should this be a subsection of `Canonical tx Mapping` I am a bit lost in the document.

Reply:
Yes, it became that after removing the section above.